src/js/player.js

```diff
@@ -3015,10 +3015,10 @@ Player.prototype.handleVolumeChange_; // eslint-disable-line
  *
  * @event error
  */
-Player.prototype.handleError_ = Player.prototype.handleError_;
+Player.prototype.handleError_; // eslint-disable-line
```

Comment:
how does this work and is it needed? seems like it is never used from a quick code search